src/app/features/collectibles/components/bitcoin/ordinals.tsx

```diff
@@ -19,7 +19,7 @@ export function Ordinals() {
         ordinals_count: inscriptionsLength,
       });
     }
-    void analytics.identify({ ordinals_count: inscriptionsLength });
+    void analytics.identify(undefined, { ordinals_count: inscriptionsLength });
```

Comment:
Weird we need to do this? 🤔

Reply:
@camerow this is your change, not sure here?